src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -136,7 +136,7 @@
  * <ul>
  * <li>
  * Property {@code customImportOrderRules} - Specify list of order declaration customizing by user.
```

Comment:
after very long conversation with @rnveach , we agreed to keep "" as value, but we need to remove `list` from description to avoid confusion

please update to
`Specify format of order declaration customizing by user.`

Reply:
moved to https://github.com/checkstyle/checkstyle/pull/7894#discussion_r407070736